File: yunva17/프로그래머스/Lv1/가운데_글자_가져오기.js
function solution(s) {
  var answer = '';
  var slength = s.length;

  if (s.length % 2 === 0) {
    answer = s[slength / 2 - 1] + s[slength / 2];
  } else {
    answer = s[Math.floor(slength / 2)];
  }

  return answer;
}

File: overthename/Programmers/level1/가운데글자가져오기.java
import java.util.*;
class Solution {
	public String solution(String s) {
		String answer = "";

		String[] list = s.split("");

		int length=(list.length-1)/2;


		if(list.length%2==0){
			for(int i=length;i<=length+1; i++){
				answer+=list[i];}
		}else{
			answer=list[(list.length-1)/2];
		}


		return answer;
	}
}